tests/core/conversations_worker/integration/test_claiming_concurrency.py

```
@@ -0,0 +1,345 @@
+
+
+def _iso_cluster(prefix: str) -> str:
+    """A synthetic cluster id unique per run — no worker watches it."""
+    return f"claimtest-{prefix}-{uuid.uuid4().hex[:12]}"
+
+
+# ---- Conversations helpers (relay client; is_relay() satisfies RLS) ----
+
+
+def _insert_pending_conversations(
+    rc, account_id: str, cluster_id: str, count: int, updated_at: str = None
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)